content/kubermatic/main/architecture/supported-providers/kubevirt/kubevirt.en.md

```diff
@@ -9,8 +9,10 @@ weight = 7
 ## Installation
 
 ### Requirements
-KKP supports KubeVirt Operator >= 0.19.0 and the Containerized Data Importer >= v1.19.0.
-It is strongly recommended to run the latest released version.
+KubeVirt version should align with the KubeVirt cluster(infra Kubernetes cluster). Follow the [Kuberntes version compatibility](https://github.com/kubevirt/kubevirt/blob/main/docs/kubernetes-compatibility.md) and
```

Comment:
@sankalp-r please change to:

Kubernetes cluster where KubeVirt is installed (KubeVirt infrastructure cluster) must be in range of [supported KKP Kubernetes clusters](https://docs.kubermatic.com/kubermatic/v2.21/tutorials-howtos/operating-system-manager/compatibility/#kubernetes-versions). The strong recommendation is to use the latest supported version.

The infrastructure cluster must have installed:
* KubeVirt - >= 0.56 and support the selected Kubernetes version.
* Containerized Data Importer - recommended the latest that supports the selected Kubernetes version.

Visit [KubeVirt compatibilty page](https://docs.kubermatic.com/kubermatic/v2.21/tutorials-howtos/operating-system-manager/compatibility/#kubernetes-versions) to find out which version of KubeVirt you can install on your infrastructure cluster.


I realized that we have to support >= v0.56.0 because of instancetype renaming.

Reply:
Don't we also need KubeVirt >= 0.57.0 for TopologySpreadConstraints ?

Reply:
+1, we need